CMakeLists.txt

```diff
@@ -87,3 +87,7 @@ get_filename_component(TOOLSET_ROOT_DIR "${TOOLSET_ROOT_DIR}" DIRECTORY) # $\VC\
 set(TOOLSET_LIB "${TOOLSET_ROOT_DIR}/lib/${VCLIBS_X86_OR_X64}")
 
 add_subdirectory(stl)
+
+# Since we don't actually build any of the tests with CMake, enable testing unconditionally

```

Comment:
this comment is not that helpful.....

enable_testing() doesn't really have much to do with building tests

Reply:
Normally you would wrap the `enable_testing()` in an `if (BUILD_TESTING)` (or would just have an `include(CTest)`) because there's a bunch of overhead to each call of `add_test()` that can be avoided if you don't actually want testing (i.e. it does nothing if `enable_testing()` hasn't been called). However, since we're not actually configuring any test binaries to be built and the amount of work that is done to enable the testing is minimal at configuration time there is no reason not to just unconditionally enable it.

It is kind of weird that we always call `enable_testing` so I do want a comment. Would it make more sense to you if I changed build to configure?